### ISSUE TITLE
Add  <category android:name="android.intent.category.BROWSABLE"/> to SEND action

### ISSUE THREAD

**Issue by PaulKinlan** [intrications/intent-intercept#32]
Hey,

I am coming up with some ways to invoke sharing from the web by creating and intent: url that has an action of SEND, right now next to 0 apps include category browsable in their manifest for the SEND action and I need a way to debug it and it would be great if this app had browsable on the url.

intent:#Intent;action=android.intent.action.SEND;S.android.intent.extra.TEXT=https%3A%2F%2Fpaul.kinlan.me%2F;S.browser_fallback_url=https:%3A%2F%2Ftwitter.com%2Fintent%2Ftweet;end

**Comment by intrications** [comment]
@PaulKinlan 

Read your blog post about this the other day so happy to help.

Until your blog post, I've never understood what android.intent.category.BROWSABLE was meant to do.

I've pushed version 2.2.3 to the Play Store. Tested and it works with the links on https://paul.kinlan.me/sharing-natively-on-android-from-the-web/. Offers me a choice between opening Intent Intercept and Plaid (https://github.com/nickbutcher/plaid).
